### DISCUSSION TITLE
Image and label for sharing website link

### DISCUSSION THREAD

**Discussion by ksw9** [greenelab/lab-website-template#147] [General] [answered]
Hi all,

Thank you for a fantastic tool! I'm wondering how to access and edit the image and text that's shared when you share a link via text/iMessage? Currently, when I share a test link, the text 'Lab Website Template' with the default testtube image is shared, with smaller text saying Home| 'My lab name'.

Thank you again and best,
Katie

**Comment by vincerubinetti** [comment, marked as answer]
Hi Katie,

For the image, you need to replace `favicons/share-thumbnail.jpg` with your own logo/text. It's just an image (which should be high res and roughly 2:1 aspect ratio).

https://github.com/greenelab/lab-website-template/wiki/Basic-Editing#favicons

The text below the image should match `title` in `_config.yaml`, and include the name of the page you linked (this is standard practice).

**Reply by liviuspenter** [reply]
Is this still the most up-to-date information? I can't see [/_includes/favicons.html](https://github.com/greenelab/lab-website-template/blob/main/_includes/favicons.html)

**Reply by vincerubinetti** [reply]
If you are on version > 1.0.0, this changed to `images/share.jpg`. The new documentation is here:

https://greene-lab.gitbook.io/lab-website-template-docs/introduction/overview

And the relevant page:

https://greene-lab.gitbook.io/lab-website-template-docs/basics/use-your-logo

**Comment by ksw9** [comment]
Got it, thank you Vincent! This is a fantastic resource!

From: Vincent Rubinetti ***@***.***>
Date: Tuesday, December 20, 2022 at 9:01 AM
To: greenelab/lab-website-template ***@***.***>
Cc: Katharine S. Walter ***@***.***>, Author ***@***.***>
Subject: Re: [greenelab/lab-website-template] Image and label for sharing website link (Discussion #147)

Hi Katie,

For the image, you need to replace favicons/share-thumbnail.jpg with your own logo/text. It's just an image (which should be high res and roughly 2:1 aspect ratio).

https://github.com/greenelab/lab-website-template/wiki/Basic-Editing#favicons

The text below the image should match title in _config.yaml, and include the name of the page you linked (this is standard practice).
